client/homebrew/utils/versionHistory.js

```diff
@@ -11,28 +13,33 @@ const DEFAULT_HISTORY_SAVE_DELAYS = {
 	'5' : 2 * 24 * 60
 };
 
-const DEFAULT_GARBAGE_COLLECT_DELAY = 28 * 24 * 60;
+const HB_DB = 'HOMEBREWERY-DB';
+const HB_STORE = 'HISTORY';
 
-const HISTORY_SAVE_DELAYS = global.config?.historyData?.HISTORY_SAVE_DELAYS ?? DEFAULT_HISTORY_SAVE_DELAYS;
-const GARBAGE_COLLECT_DELAY = global.config?.historyData?.GARBAGE_COLLECT_DELAY ?? DEFAULT_GARBAGE_COLLECT_DELAY;
+const DEFAULT_GARBAGE_COLLECT_DELAY = 28 * 24 * 60;
 
+let HISTORY_SAVE_DELAYS = DEFAULT_HISTORY_SAVE_DELAYS;
+let GARBAGE_COLLECT_DELAY = DEFAULT_GARBAGE_COLLECT_DELAY;
 
 
 function getKeyBySlot(brew, slot){
+	// Return a string representing the key for this brew and history slot
 	return `${HISTORY_PREFIX}-${brew.shareId}-${slot}`;
 };
 
-function getVersionBySlot(brew, slot){
-	// Read stored brew data
-	// - If it exists, parse data to object
-	// - If it doesn't exist, pass default object
-	const key = getKeyBySlot(brew, slot);
-	const storedVersion = localStorage.getItem(key);
-	const output = storedVersion ? JSON.parse(storedVersion) : { expireAt: '2000-01-01T00:00:00.000Z', shareId: brew.shareId, noData: true };
-	return output;
-};
-
-function updateStoredBrew(brew, slot = 0) {
+// function getVersionBySlot(brew, slot){
+// 	// Read stored brew data
+// 	// - If it exists, parse data to object
+// 	// - If it doesn't exist, pass default object
+// 	const key = getKeyBySlot(brew, slot);
+// 	const storedVersion = IDB.get(key);
+// 	const output = storedVersion ? storedVersion : { expireAt: '2000-01-01T00:00:00.000Z', shareId: brew.shareId, noData: true };
+// 	return output;
+// };
```

Comment:
Looks like we can delete this block

Reply:
Yes, I commented it out in case I broke something by no longer having it available, but it should be fine to just delete now.
It will be saved in a reversible commit anyway.